examples/metatensor-plumed/lj-crystal.py

```diff
@@ -0,0 +1,543 @@
+# deep local minimum. Let's load them and visualize all three (our starting
+# structure and the two targets) using `chemiscope`.
+
+minimal = ase.io.read("lj-oct-0k.xyz")
+# FIXME: this does not look like the "other" stable structure
+other = ase.io.read("lj38.xyz")
+
+settings = {"structure": [{"playbackDelay": 50, "unitCell": True, "bonds": False}]}
+chemiscope.show([minimal, other, atoms], mode="structure", settings=settings)
+
+
+# %%
+# Defining our custom collective variable
+# ---------------------------------------
+#
+# To distinguish between the liquid-like state and the highly ordered
+# face-centered cubic (FCC) packing of the global minimum, we use **Steinhardt
+# order parameters**, specifically :math:`Q_4` and :math:`Q_6`. These parameters
+# are rotationally invariant and measure the local orientational symmetry around
+# each atom.
+#
+# - :math:`Q_6` is often high for both icosahedral and FCC-like structures,
+#   making it a good measure of general "solidness".
+# - :math:`Q_4` helps to distinguish between different crystal packing types. It
+#   is close to zero for icosahedral structures but has a distinct non-zero value
+#   for FCC structures.
+#
+# We will build a model that calculates global, system-averaged versions of these
+# parameters.
+
+
+# %%
+#
+# Encapsulating the Logic in a ``torch.nn.Module``
+# '''''''''''''''''''''''''''''''''''''''''''''''''
+#
+# To make this CV usable by PLUMED via the ``METATOMIC`` interface, we must wrap
+# our calculation logic in a ``torch.nn.Module``. This class takes a list of
+# atomic systems and returns a `metatensor.TensorMap` containing the calculated
+# CV values. The interface is defined in [TODO link].
+
+
+class CollectiveVariable(torch.nn.Module):
+    def __init__(self, cutoff, angular_list):
+        super().__init__()
+
+        self.max_angular = max(angular_list)
+        # initialize and store the featomic calculator inside the class
+        self.spex = featomic.torch.SphericalExpansion(
+            **{
+                "cutoff": {
+                    "radius": 1.3,
+                    "smoothing": {"type": "ShiftedCosine", "width": 0.5},
+                },
+                "density": {"type": "Gaussian", "width": 0.3},
+                "basis": {
+                    "type": "TensorProduct",
+                    "max_angular": 6,
+                    "radial": {"type": "Gto", "max_radial": 3},
+                },
+            }
+        )
+        self.selected_keys = mts.Labels(
+            "o3_lambda", torch.tensor(angular_list).reshape(-1, 1)
+        )
+
+    def forward(
+        self,
+        systems: List[mta.System],
+        outputs: Dict[str, mta.ModelOutput],
+        selected_atoms: Optional[mts.Labels],
+    ) -> Dict[str, mts.TensorMap]:
+
+        # execute the same code as above
+        spex = self.spex(
+            systems, selected_samples=selected_atoms, selected_keys=self.selected_keys
+        )
+
+        if len(spex) == 0:
```

Comment:
```suggestion
        if len(systems) == 0:
```

this might be clearer